packages/@aws-cdk/aws-stepfunctions-tasks/test/codebuild/batch-get-reports.test.ts

```diff
@@ -0,0 +1,62 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Stack } from '@aws-cdk/core';
+import { CodeBuildBatchGetReports } from '../../lib/codebuild/batch-get-reports';
+
+let stack: Stack;
+let reportArns: string[];
+
+beforeEach(() => {
+  //GIVEN
+  stack = new Stack();
+  reportArns = ['reportArn1', 'reportArn2'];
```

Comment:
add a test that uses a `path` to verify that `ReportArns.$` gets rendered as we would expect.